static/src/deploys-radiator/app/main.css

```diff
@@ -71,3 +75,9 @@ a {
 a:hover {
     text-decoration: underline;
 }
+
+div > div {
+    float: left;
+    width: 31%;
+    margin: 0 1%;
+}
```

Comment:
I've been using flexbox but the browser on the Pi doesn't support it. We need to try and get Firefox/Chrome on there :tongue: